rasa/core/policies/form_policy.py

```diff
@@ -17,6 +17,8 @@
 
 logger = logging.getLogger(__name__)
 
+# TODO: Remove and add deprecation error
```

Comment:
@Ghostvv I'd actually do that right now?

Reply:
without `remove` though 😅

yes, and we need to add deprecation error to all other rule-like policies except memo